composer.json

```diff
@@ -19,7 +19,7 @@
     "require": {
         "php": "^7.0",
```

Comment:
Maybe also change this to `^7.2` as it is in `shopware/shopware`. What do you think?